integration-tests/graal/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+  id("otel.java-conventions")
+  id("java-library")
+  id("org.graalvm.buildtools.native") version "0.9.14"
```

Comment:
We centrally manage plugin versions in this repo in the `pluginManagement` block of `settings.gradle.kts`. Can you move this id / version there and omit the version here?

```
pluginManagement {
  plugins {
    ....
    id("org.graalvm.buildtools.native") version "0.9.14"
  }
}
```